src/interpreter/vm/VM.cpp

```diff
@@ -0,0 +1,160 @@
+#include <VM.h>
+#include <opcode.h>
+#include <cstdlib>
```

Comment:
Maybe we want to reorganize these? For instance we could do standard library in one section and our headers in another (that's what I've been doing) or alphabetical.